jmclient/jmclient/blockchaininterface.py

```diff
@@ -215,6 +215,39 @@ def get_block(self, blockheight):
             return False
         return block
 
+    def rescanblockchain(self, start_height, end_height=None):
```

Comment:
```suggestion
    def rescanblockchain(self, start_height: int, end_height: Optional[int] = None) -> None:
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1461/commits/7f4eaa9846f2f489480de1f2f8bfa9a1e2e96686